_source/_posts/2023-07-31-workflows-workshop.md

```diff
@@ -0,0 +1,939 @@
+Workflows can create non-activated accounts in all apps one week before a new employee's start date and then activate them on their first day.
+
+If an employee leaves your company, Workflows can deactivate the user account, remove their access to all apps except payroll, and then delete the account after a year.
+
+### Send notifications for lifecycle events
+For a lifecycle event such as an app assignment or user suspension, Workflows can notify your IT team through email or Slack.
+
+### Log and share lifecycle events
+Workflows can query Okta APIs and System Log events, run logic, and even compile data into a CSV file. Then, Workflows can email that file to your teams.
+
+This is a short list of what is possible.
+
+### Workflows flows
+In Workflows, you will create flows. A flow is a sequence of steps to complete a goal. It's similar to a script where every code line is a step to complete a goal. In Workflows, you will build the flow visually without writing any code. 
+
+You will learn about the Workflows building blocks in a few minutes. First, let's create your first flow. 
+
+## Getting access to Workflows
+There are two ways to access Workflows. 
+
+1. You might be entitled to Workflows if you already use some Okta products
+2. Access to Workflows as part of WIC (Workforce Identity Cloud) trial
+
+### Workflows when using other Okta products
+If you have Okta SSO or Okta UD you might be entitled to Workflows. Check if you can access Workflows by going to **Okta organization > Admin > Workflows > Workflows console**. If you use these products but don't have access to Workflows, please contact your account manager. 
+
+### Workflows as part of WIC trial
+To sign up for a Workforce Identity Cloud trial:
+
+1. Go to the [Okta Workforce trial page](https://www.okta.com/free-trial/workforce-identity/)
+2. Register for a trial to access Workflows
+
+### Opening the Workflows console
+
+To access Workflows:
+
+1. Sign-in into your Okta admin dashboard
+    * If you are using a trial account, the admin dashboard link will be in the following format: `https://trial-xxxxxxxx-admin.okta.com/admin/dashboard`
+2. On the right-hand side menu, click **Workflows > Workflows console** to open Workflows
+
+
+You will see the Workflows home page:
+
+{% img blog/workflows-workshop/Workflows_console_main.jpg alt:"Workflows home page" %}{: .center-image }
+
+## Steps to build automated email report using Okta Workflows
+In this workshop, you will learn how to build an automated email report in Okta Workflows. 
+
+You will learn and complete the following steps: 
+
+- Building an Okta Workflows flow and automation
+- Scheduling the flow to run periodically
+- Enhancing an application with a new API service
+- Calling the API service to retrieve todos items from an application
+- Creating a todo summary and sending the summary via email
+- Testing the flow
+
+> 
+This workshop uses a locally deployed application and its API.
```

Comment:
Is this on the wrong line? (inadvertent new line?)

Reply:
Deleted.